app/controllers/edit-service-name.controller.js

```diff
@@ -40,12 +40,11 @@ exports.post = (req, res) => {
     })
     res.redirect(formatPath(paths.editServiceName.index, req.service.externalId))
   } else {
-    return serviceService.updateServiceName(serviceExternalId, serviceName, serviceNameCy, correlationId)
-      .then(() => {
-        res.redirect(paths.serviceSwitcher.index)
-      })
-      .catch(err => {
-        responses.renderErrorView(req, res, err)
-      })
+    try {
+      await serviceService.updateServiceName(serviceExternalId, serviceName, serviceNameCy, correlationId)
+      res.redirect(paths.serviceSwitcher.index)
+    } catch (err) {
+      responses.renderErrorView(req, res, err)
```

Comment:
I think passing `err` to `renderErrorView` will try to use it as a message to display to the user. But in this case it might end up as something like `[Object object]` or something shown to the user 😬. Even if not, we shouldn't be displaying error messages like this to the user.

Can we change this so we call `next` and pass the error to it. The new error handling middleware code at the end of the stack will then handle it and call the error.

I know this isn't something you changed, but we should fix it.

Reply:
Done